tests/conftest.py

```diff
@@ -6,13 +9,45 @@
 pytest_plugins = "sphinx.testing.fixtures"
 
 
+def _set_localconf(app):
+    """
+    Load additional configuration for local environment
+
+    example: roots/test-root/localconf.json
+    {
+      "drawio_binary_path": "/Applications/draw.io.app/Contents/MacOS/draw.io",
+      "drawio_headless": false
+    }
+    """
+
+    local_conf_path = os.path.join(
+        os.path.dirname(__file__),
+        'roots',
+        'test-root',
+        'localconf.json',
+    )
+
+    try:
+        pass
+        with open(local_conf_path, 'r') as f:
+            d = f.read()
+            conf = json.loads(d)
```

Comment:
```suggestion
            conf = json.loads(d)
            app.config.update(conf)
```

Reply:
`update()`?

Reply:
app.config is not dict. It is Config class in sphinx/config.py.
Config class has no update method.

Reply:
My bad.